### ISSUE TITLE
Add a non-watch generate command

### ISSUE THREAD

**Issue by joshuacerbito** [wyldstyle/wyldstyle#4]
Create a non-watch command that can be called to generate the utility classes

**Comment by jaggy** [comment]
Not sure how the api should be here though. I kinda want to generate the styles by default.

```
ws file1 file2 file3 output.css # by default generates the output file

ws file1 file2 file3 output.css --watch
```

Thoughts?

**Comment by jaggy** [comment]
Implemented on PR #5
